main.go

```diff
@@ -196,10 +218,33 @@ func main() {
 						Name:        "proof-file",
 						Destination: &proofFile,
 					},
+					&cli.StringFlag{
+						Name:        "message",
+						Destination: &message,
+						DefaultText: "Hello World!",
+					},
 				},
 				Name: "verify-rln-proof",
 				Action: func(cCtx *cli.Context) error {
-					err := VerifyRlnProof(cfg, proofFile)
+					err := VerifyRlnProof(cfg, proofFile, message)
+					return err
+				},
+			},
+
+			{
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:        "membership-file",
+						Destination: &membershipFile,
+					},
+					&cli.StringFlag{
+						Name:        "message",
+						Destination: &message,
+					},
+				},
+				Name: "send-message",
+				Action: func(cCtx *cli.Context) error {
+					err := SendMessage(cfg, membershipFile, message)
```

Comment:
This is mostly a nitpick, but consider changing the signature of `SendMessage` to this:
```go
func SendMessage(ctx context.Context, cfg *Config, membershipFile string, message string) error
```
And then passing the context from the application in here:
```go
err := SendMessage(cCtx.Context, cfg, membershipFile, message)
```

So you can avoid having to use `context.Background()` in 751 and 812 and use instead the context you receive as a parameter